wagtail_localize/operations.py

```diff
@@ -33,7 +33,7 @@ def create_translations(self, instance, include_related_objects=True):
             return
         self.seen_objects.add(instance.translation_key)
 
-        source, created = TranslationSource.get_or_create_from_instance(instance)
+        source, _ = TranslationSource.get_or_create_from_instance(instance)
```

Comment:
I'd suggest using `_created` to avoid a scenario where we import gettext/gettext_lazy as _

Reply:
Good point, reverted in the latest commit 👍